src/main/java/seedu/dictionote/model/ModelManager.java

```diff
@@ -264,6 +268,14 @@ private Note combineNote(Note firstNote, Note secondNote) {
         return combinedTag;
     }
 
+    @Override
+    public void convertTxtNote(Note note) throws IOException {
+        String noteToString = note.toString();
+        BufferedWriter writer = new BufferedWriter(new FileWriter(FILE_PATH + note.createFileName()));
+        writer.write(noteToString);
+        writer.close();
+    }
```

Comment:
I suggest doing this in a separate class. Model should not be responsible to be writing to file.

Reply:
I have moved it inside the command itself